h/views/api/config.py

```diff
@@ -55,6 +73,8 @@ def add_api_view(
     settings.setdefault("renderer", "json")
     settings.setdefault("decorator", cors_policy)
 
+    versions = versions or [API_VERSION_DEFAULT]
```

Comment:
As I mentioned in my previous WIP PR on this stuff, this param is currently optional but won't be for long.